src/version.rs

```diff
@@ -70,9 +72,115 @@ pub struct VersionLinks {
     pub dependencies: String,
     pub version_downloads: String,
     pub authors: String,
+    pub build_info: String,
+}
+
+#[derive(Insertable, AsChangeset, Identifiable, Queryable, Associations)]
+#[belongs_to(Version)]
+#[table_name = "build_info"]
+#[primary_key(version_id, rust_version, target)]
+pub struct BuildInfo {
```

Comment:
Could we have a doc comment here as well as for all structs, enums, and functions in this file?